pennylane_runtime/vqe_runtime.py

```diff
@@ -0,0 +1,193 @@
+# Copyright 2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+This module contains a custom VQE runtime program that can be uploaded to IBMQ.
+"""
+# pylint: # pylint: disable=too-many-arguments,too-many-branches,too-many-statements
+
+import numpy as np
+import scipy.optimize as opt
+from scipy.optimize import OptimizeResult
+import mthree
+
+import qiskit.circuit.library.n_local as lib_local
+from qiskit.algorithms.optimizers import SPSA, QNSPSA
+from qiskit import QuantumCircuit, transpile
+
+
+def opstr_to_meas_circ(op_str):
+    """Takes a list of operator strings and makes circuit with the correct post-rotations for measurements.
+
+    Args:
+        op_str (list): List of strings representing the operators needed for measurements.
```

Comment:
Would be good to have an example of how this operator string representation looks like.